### PR TITLE
issue/58 tooltip to open when focused

### PR DESCRIPTION
**Pull request by CezCz**
Tooltip to show it's content when focused with tab button.

## Description
Vent doesn't handle multiple events, I moved two events that were trying to register in one call to two calls.
This helped for tooltip to open, but as it's focused by default, it triggered focusout on the icon causing this tooltip to close. I turned off autofocus on open, since there are no interactable elements in the tooltip.

My PR deletes redundant spaces cluttering the PR, the interesting parts are here:

https://github.com/adobe/coral-spectrum/compare/issue/58?expand=1#diff-ef943d799f5de6765589327dc5a4f795R93
https://github.com/adobe/coral-spectrum/compare/issue/58?expand=1#diff-ef943d799f5de6765589327dc5a4f795R309
https://github.com/adobe/coral-spectrum/compare/issue/58?expand=1#diff-7afc3cb344f58628908dfa726733a423R56

## Related Issue
https://github.com/adobe/coral-spectrum/issues/58

## Motivation and Context
Tooltip is accessible from keyboard

## How Has This Been Tested?

Manual tests and automated test, also checked in examples

Chrome  80.0.3987.163
Firefox 74.0

Could potentially affect examples, checked and it does not.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: coral-component-tooltip/src/scripts/Tooltip.js [hunk 1]
```diff
@@ -282,36 +283,55 @@ class Tooltip extends Overlay {
...
+  _handleOpenTooltip() {
+    return () => {
       // Don't let the tooltip hide
       this._cancelHide();
-      
+
       if (!this.open) {
         this._cancelShow();
-        
+        this.ariaDescribedby = this._contentId;
```
Comment (icaraps) [inline coral-component-tooltip/src/scripts/Tooltip.js:333]: What is `this._contentId` ?

Comment (CezCz) [reply coral-component-tooltip/src/scripts/Tooltip.js]: Trash, removed

**Comment by icaraps** [comment]
2 tiny things: 
* you can replace `this.focusOnShow = Overlay.focusOnShow.OFF;` with `this._focusOnShow = Overlay.focusOnShow.OFF;` it's subtle but it's faster
* `this._handleOpenTooltip` is a function that returns an arrow function. Is it possible instead of calling the function to simply pass the function and remove the `return` ? Not sure if it makes sense ...

**Comment by CezCz** [comment]
@icaraps Please check the changes. With returning lambda, it's just a question of how to retain this.

**Pull request merged**